src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
```

Comment:
I extracted this from `Atom::saveSync()`, which didn't wrap it in a `try/catch`. What would the `try/catch` be for in this case?